src/test/app/drafts/models/address.ts

```diff
@@ -1,8 +1,118 @@
 import { expect } from 'chai'
 
-import { Address } from 'app/forms/models/address'
+import { Address, ValidationErrors } from 'app/forms/models/address'
+import { Address as ClaimAddress } from 'claims/models/address'
+import { Validator } from 'class-validator'
+import { expectValidationError } from '../../forms/models/validationUtils'
+import * as randomstring from 'randomstring'
 
 describe('Address', () => {
+  describe('validation', () => {
```

Comment:
I do not think this is appropriate place for validation tests. We already have validation tests in `src/test/app/forms/models/address.ts` and these tests should have been amended.